src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -20,29 +19,20 @@ internal enum Stamp
             SetToMaxUsed = 3,
         }
 
-        internal int _version;
         private List<Cookie> _list = new List<Cookie>();
 
         private DateTime _timeStamp = DateTime.MinValue;
         private bool _hasOtherVersions;
 
-        private readonly bool _isReadOnly;
-
         public CookieCollection()
         {
-            _isReadOnly = true;
-        }
-
-        internal CookieCollection(bool IsReadOnly)
```

Comment:
Also, the `IsReadOnly` property isn't used anywhere in reference source that I could find. And the collection is obviously not read-only (even though, the default constructor sets the value to `true`). I'm not sure what its purpose is.